apollo-router/src/configuration/mod.rs

```diff
@@ -1255,3 +1265,16 @@ fn default_graphql_path() -> String {
 fn default_graphql_introspection() -> bool {
     false
 }
+
+/// Configuration for Batching
+#[derive(Debug, Default, Clone, Deserialize, Serialize, JsonSchema)]
+#[serde(deny_unknown_fields)]
+#[serde(default)]
+pub(crate) struct Batching {
+    /// Activates Batching (disabled by default)
+    pub(crate) enabled: bool,
+
+    /// Batching mode (batch_http_link is supported)
+    #[serde(default)]
```

Comment:
This is useless as the struct is already tagged with `serde(default)`

Reply:
Yup. I changed this to match the spec change, so neither the struct or the member are default now.